tests/functional/comment/__init__.py

```diff
@@ -116,7 +116,7 @@ def teardown_package():
     global TEST_WORKSPACE
 
     print("Removing: " + TEST_WORKSPACE)
-    shutil.rmtree(TEST_WORKSPACE)
+    shutil.rmtree(TEST_WORKSPACE, ignore_errors=True)
```

Comment:
I don't think we should try and save the world here in this pull request. The entire mechanism of which code runs with which environment needs to be revised to properly fix this particular issue. (Which, as I said, I've already done.) Until then, if the tests fail due to the comment not shutting down in the ideal order, we can always just mash "restart" on Travis. This change brought in much more exotic flows to error.